aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Test/WeatherApiTest.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.AspNetCore.Http.HttpResults;
+
+namespace Tests
+{
+    [TestClass()]
+    public class WeatherApiTests
+    {
+        [TestMethod()]
+        public void MapWeatherApiTest()
+        {
+            var result = WeatherApi.GetAllWeathers();
+            // assert that the result is a typed result of type WeatherForecast[]
+            Assert.IsInstanceOfType(result, typeof(Ok<WeatherForecast[]>));
+        }
+      
+    }
+
+}
```

Comment:
Git hub like a blank line at the end of a file
![image](https://user-images.githubusercontent.com/3605364/179326772-2ba3977b-6bd8-44ad-b3ee-ee45996f9416.png)
```suggestion
}

```